src/components/NavBar.tsx

```diff
@@ -1,50 +1,71 @@
-import * as React from 'react'
+// @refresh reset
```

Comment:
Forcing the state to be refreshed on page load is likely completely redundant/useless. What is the purpose of this explicit state refresh?

Context about the line of code: https://reactnative.dev/docs/fast-refresh#tips

Reply:
See my previous comments about the navbar styling (and icon) which are part of my review.